src/OpenTelemetry/Trace/Samplers/AlwaysOnActivitySampler.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <copyright file="AlwaysOnActivitySampler.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace OpenTelemetry.Trace.Samplers
+{
+    /// <summary>
+    /// Sampler implementation which will sample in all the spans.
```

Comment:
nit: I tend to find "sample in" and "sample out" ambiguous, and use "sample" to mean record/generate.
```suggestion
    /// Sampler implementation which will always sample, i.e. record, all the spans.
```

Reply:
Hum, I just remembered that OTel makes a distinction between sampled and recorded my wording with "record" is confusing in this case...

Reply:
- Java uses: "always makes a "yes" decision on {@link Span} sampling."
- Go uses: "returns a Sampler that samples every trace."

I prefer the latter.